### PR TITLE
Convert filters form from POST to GET

### PR DESCRIPTION
**Pull request by tim-wilde-nimble**
Convert filters form from POST to GET to avoid warnings using browser navigation controls.

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecome.Data.Tests/Services/AcademyConversionProjectRepositoryTests.cs [hunk 1]
```diff
@@ -79,7 +72,7 @@ public void GivenDeliveryOfficers_GetsNoProjects_WhenDeliveryOfficerHasNoneAssig
 			_subject = new AcademyConversionProjectRepository(new MockHttpClientFactory(_messageHandler));
 			var deliveryOfficers = new List<string>() { "John Smith" };
 
-			Assert.Equal(0, _subject.GetAllProjects(1, 50, deliveryOfficers, string.Empty, string.Empty).Result.Body.Data.Count());
+			Assert.Equal(0, _subject.GetAllProjects(1, 50, string.Empty, default, deliveryOfficers).Result.Body.Data.Count());
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecome.Data.Tests/Services/AcademyConversionProjectRepositoryTests.cs:75]: Couple of warnings here.

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectList/Index.cshtml.cs [hunk 2]
```diff
@@ -40,41 +39,25 @@ public IndexModel(IAcademyConversionProjectRepository repository)
...
+			Filters.AvailableDeliveryOfficers = allResponses.Body.Data.Select(x => x.AssignedUser.FullName).Where(s => string.IsNullOrEmpty(s) is false).Distinct().ToList();
+
+			ApiResponse<List<string>> statusesResponse = await _repository.GetAvailableStatuses();
+			if (statusesResponse.Success) Filters.AvailableStatuses = statusesResponse.Body.ConvertAll(r => r.SentenceCase());
+
 			if (CurrentPage - 5 > 1)
+			{
 				StartingPage = CurrentPage - 5;
-
-			return Page();
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectList/Index.cshtml.cs:77]: This is interesting, are we not required to return a Page() in the OnGet?

Comment (tim-wilde-nimble) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectList/Index.cshtml.cs:77]: Nope; the default 'result' is to render the associated template.

File: ApplyToBecomeInternal/ApplyToBecome.Data.Tests/Services/AcademyConversionProjectRepositoryTests.cs (hunk 1 above)
Comment (tim-wilde-nimble) [reply ApplyToBecomeInternal/ApplyToBecome.Data.Tests/Services/AcademyConversionProjectRepositoryTests.cs]: Gave these tests a bit of an update!

**Review by jordan-beard — APPROVED** [review]

**Review by dneed-nimble — APPROVED** [review]

**Pull request merged**